demisto_sdk/commands/lint/linter.py

```diff
@@ -1560,6 +1569,8 @@ def _get_docker_images_for_lint(
             script_obj (dict): A yml dict of the integration/script that lint runs on.
             script_id (str): The ID of the integration/script that lint runs on.
             docker_image_flag (str): A flag indicates on which docker images lint should run.
+            docker_image_target (Optional[str]): The docker image to test native supported content on.
```

Comment:
I think it is worth adding an exception in case a user selects the 'native:target' flag without setting a docker_image_target or setting a docker_image_target with another flag.